pkg/apis/management.cattle.io/v3/machine_types.go

```diff
@@ -415,3 +416,8 @@ type S3CredentialConfig struct {
 	DefaultBucket        string
 	DefaultFolder        string
 }
+
+type AlibabaCredentialConfig struct {
+	AccessKeyId     string `json:"accessKeyId" norman:"required"`
```

Comment:
```suggestion
	AccessKeyID     string `json:"accessKeyId" norman:"required"`
```
https://go.dev/wiki/CodeReviewComments#initialisms